src/components/Wizard/CreateVmWizard/BasicSettingsTab.js

```diff
@@ -38,6 +35,7 @@ import {
   PROVISION_SOURCE_CONTAINER,
   PROVISION_SOURCE_URL,
   PROVISION_SOURCE_IMPORT,
+  PROVISION_SOURCE_IMAGE,
```

Comment:
Image is a pretty vague term and I suggest we avoid it.  Do you mean we are cloning an existing vm disk?  Are we attaching it (taking ownership of a PVC)?  For clones I'd suggest PROVISION_SOURCE_CLONE_DISK.  For assuming ownership I'd recommend using PROVISION_SOURCE_ATTACH_DISK.

Reply:
We are cloning all PVS right now (attach disk & image). So we should also add clone keyword to attach disk as well - to be consistent.

Reply:
@aglitke @matthewcarleton 

What is the best solution?
1. use IMAGE 
2. put CLONE everywhere in the UI (e.g. button with `Attach Cloned Disk`)
3. use CLONE only in PROVISION_SOURCE_CLONE_DISK option. This would make the UI inconsistent

Reply:
@aglitke maybe you can shed some more light here but if we are cloning regardless of attach or clone maybe we can just keep it as "provision source: disk" and allow them to upload or choose from existing? @suomiy in the designs for the storage step we've updated the action to just "add disk" rather than create/attach just to simply it, which I think aligns with what I'm saying.

Reply:
@suomiy can you clarify what you mean here: 
>We are cloning all PVS right now (attach disk & image). So we should also add clone keyword >to attach disk as well - to be consistent.

Reply:
- Attaching disk by clicking `Attach Disk` in `Storage` tab clones the underlying PVC and then attaches it to the VM
- Choosing provision source `Image` (current impl) clones the underlying PVC and then the VM boots from it

I would prefer to have consistent UI and have the same terminology everywhere:

- either we decide it is important to pass the clone information to the user, then we should point this out everywhere in the UI
- or we decide it is not important and we omit that in the UI

I hope it makes sense

Reply:
@suomiy so can you confirm that we are updating the storage section to show `add disk` rather  two separate actions of `create` and `attach`. I think this will help simplify the problem you are describing.

Reply:
I guess yes it will happen eventually (not that soon though). Can we merge this PR and resolve this once we start implementing the new flow?

Reply:
Attach and clone are two separate operations.  I'd expect attach to directly use an existing pvc without cloning it.  Clone should create a clone of an existing PVC.  There are use cases for both of these.

Reply:
@aglitke Can you explain more why a user would Attach in one case and Clone in another? I was going to suggest we hide the term Clone from the user since it's really the underlying implementation and in the end it's just pointing to a PVC, but this actually makes a big difference :)

Reply:
I created an issue for this conversation as it is outside of the scope of this PR: https://github.com/kubevirt/web-ui-components/issues/283